### PR TITLE
Scheduled Vuneraibilities workflow simplification

### PR DESCRIPTION
**Pull request by TomaszGaweda**
Use matrix instead of many jobs

### CODE REVIEW

File: .github/workflows/scheduled_vulnerability_scan.yml [hunk 1]
```diff
@@ -6,22 +6,9 @@ on:
 
 jobs:
     trigger-vulnerability-scan-5_2_z:
+        strategy:
+            matrix:
+                ref: [ '5.2.z', '5.1.z', '5.0.z', '4.2.z' ]
```
Comment (ldziedziul) [inline .github/workflows/scheduled_vulnerability_scan.yml:11]: ❤️

File: .github/workflows/scheduled_vulnerability_scan.yml [hunk 2]
```diff
@@ -6,22 +6,9 @@ on:
 
 jobs:
     trigger-vulnerability-scan-5_2_z:
```
Comment (ldziedziul) [inline .github/workflows/scheduled_vulnerability_scan.yml:8]:
```suggestion
    trigger-vulnerability-scan:
        name: Scan ${{ matrix.ref }}
```

File: .github/workflows/scheduled_vulnerability_scan.yml [hunk 3]
```diff
@@ -6,22 +6,9 @@ on:
...
-            ref: 5.0.z
-    trigger-vulnerability-scan-4_2_z:
-        uses: ./.github/workflows/vulnerability_scan_subworkflow.yml
-        with:
-            ref: 4.2.z
-    trigger-vulnerability-scan-4_1_z:
-        uses: ./.github/workflows/vulnerability_scan_subworkflow.yml
-        with:
-            ref: 4.1.z
+            ref: ${{ matrix.ref }}
```
Comment (ldziedziul) [inline .github/workflows/scheduled_vulnerability_scan.yml:14]: no new line ;)

Comment (ldziedziul) [reply .github/workflows/scheduled_vulnerability_scan.yml]: 🤣 image

**Review by frant-hartm — APPROVED** [review]
LGTM (after fixing items Lukaz pointed out)

File: .github/workflows/scheduled_vulnerability_scan.yml (hunk 3 above)
Comment (TomaszGaweda) [reply .github/workflows/scheduled_vulnerability_scan.yml]: oh come on :D Yeah, space was there xD

**Pull request merged**